subs.css

```diff
@@ -49,3 +49,7 @@
   opacity: 1.0;
   cursor: pointer;
 }
+
+input {
```

Comment:
can you please change this to `#osasoft-better-subscriptions_subs-grid {` just to be sure we're targetting only the checkbox

Reply:
sure no problem